docker-compose.platform.yml

```diff
@@ -32,21 +32,20 @@ services:
       - DPNS_TOP_LEVEL_IDENTITY=${PLATFORM_DPNS_OWNER_ID}
       - NODE_ENV=${ENVIRONMENT:?err}
       - LOGGING_LEVEL=${PLATFORM_DRIVE_ABCI_LOG_LEVEL:?err}
+      - INITIAL_CORE_CHAINLOCKED_HEIGHT=${PLATFORM_DRIVE_TENDERDASH_GENESIS_INITIAL_CORE_CHAIN_LOCKED_HEIGHT:-1}
     command: npm run abci
 
-  drive_tendermint:
-    image: ${PLATFORM_DRIVE_TENDERMINT_DOCKER_IMAGE:?err}
+  drive_tenderdash:
+    image: ${PLATFORM_DRIVE_TENDERDASH_DOCKER_IMAGE:?err}
     restart: unless-stopped
     depends_on:
       - drive_abci
     ports:
-      - ${PLATFORM_DRIVE_TENDERMINT_P2P_PORT:?err}:26656 # P2P
-      - 127.0.0.1:${PLATFORM_DRIVE_TENDERMINT_RPC_PORT:?err}:26657 # RPC
+      - ${PLATFORM_DRIVE_TENDERDASH_P2P_PORT:?err}:26656 # P2P
+      - 127.0.0.1:${PLATFORM_DRIVE_TENDERDASH_RPC_PORT:?err}:26657 # RPC
     volumes:
-      - ${MN_HOME_DIR:?err}/${CONFIG_NAME:?err}/genesis.json:/tendermint/config/genesis.json
-      - ${MN_HOME_DIR:?err}/${CONFIG_NAME:?err}/config.toml:/tendermint/config/config.toml
-      - drive_tendermint_homedir:/tendermint
-    entrypoint: sh -c "/usr/bin/tendermint init && /usr/bin/tendermint node"
+      - ${MN_HOME_DIR:?err}/${CONFIG_NAME:?err}/tenderdash:/tenderdash/config
```

Comment:
config dir should be mounted read-only?

Reply:
I guess so. When we start docker compose it must be initialized already